README.md

```diff
@@ -1,2 +1,50 @@
-# react-bookstore-shubh
-This app will keep track of books in a library 
+# Book Store
+
+> The website allows users to keep record of books and update book data.
+
+## Built With
+
+- Major languages: JavaScript, HTML, CSS
+- linters
+- Frameworks: React
+
+## Live Demo (if available)
+
+## Screenshots
+
+<hr>
+
+Getting Started
+To get the content of this project locally you need to run this command in your terminal:
```

Comment:
- [ ] [optional] Kindly check the arrangement of contents in your getting started section, it would be nice making the title bold.